com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineAssetEditor.cs

```diff
@@ -379,7 +379,7 @@ void DrawShadowResolutionTierSettings()
                 {
                     var fieldSlot = new Rect(contentRect.x + pixelShift, contentRect.y, num - labelWidth, contentRect.height); // Define the rectangle for the field
                     int value = EditorGUI.DelayedIntField(fieldSlot, values[index].intValue);
-                    values[index].intValue = Mathf.Max(128, Mathf.NextPowerOfTwo(value));
+                    values[index].intValue = Mathf.Max(UniversalAdditionalLightData.AdditionalLightsShadowMinimumResolution, Mathf.NextPowerOfTwo(value));
```

Comment:
This isn't from this change, but we really need to get away from defaulting to power of two sizes. Most modern GPU architectures support non power of two sizes, and it is only really relevant for mipmaps and compressed formats.

Reply:
If it is a atlas fitting limitation, then it is a bit more understandable.

Reply:
Yes, in the case of Additional Light shadows, it is a limitation of the current (simple) atlas fitting.